.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
Is https://github.com/ghedo/openssl/commit/fdf8e4de835ec3866fc30abcb7d06ef9ae6909da what you meant? If so no change happened
https://travis-ci.org/openssl/openssl/jobs/84121505#L2072